test/test-run-express-metrics.js

```diff
@@ -0,0 +1,34 @@
+var http = require('http');
+var tap = require('tap');
+var debug = require('./debug');
+var helper = require('./helper');
+var run = helper.runWithControlChannel;
+
+helper.pass = true; // Use tap, not this check.
```

Comment:
You'll need to add something to skip these tests when run under mocha. Currently CI is failing because it tries to run this code but `helper.runWithControlChannel` isn't defined because helper.js returns before exporting it.

Something like this near the top of the file:
```js
if (helper.skip()) return;
```